src/content/pages/sponsors.html

```diff
@@ -157,7 +157,7 @@ <h2 class="section-title h3" id="donations-label">{{ site.sponsors_page.recent_d
             {% for donation in donations | limitTo(50) %}
             <li class="donations-item">
                 {% set donation_image = donation.image %}
-                {% set donation_name = donation.name %}
+                {% set donation_name = donation.name or '' %}
```

Comment:
See https://mozilla.github.io/nunjucks/templating.html#default-value-default-boolean

Reply:
If `donation.name` never seems to go to `null` when the data is re-fetched, then it seems that we don't need to set a fallback and just wait patiently for the data to update

Reply:
Nah, it will be `null` unless the user sets the name field. It's easy to reproduce, remove your name from your github profile then fetch data with the github graphql explorer:

![](https://user-images.githubusercontent.com/1091472/209440256-f53c22e6-9a52-42fa-b7c2-1bd6b1620eb1.png)

Reply:
https://github.com/eslint/eslint.org/pull/392/files#diff-e459d8973a8d888faaadd76b2c93a38b2638df9c9e16a746c479452996b80d38L230

I mean `donation.name` will fall back to `sponsor.login` when `sponsor.name` is `null`. Considering that `login` can never be `null`, there is no need to set fallback.

Reply:
Sure, will revert this change once review is done. Keeping it now so the CI can run without errors.

Reply:
Can we update the data files in this PR instead?

Reply:
This filter was broken because of multiple same (`null`) names:

https://github.com/eslint/eslint.org/blob/415e0bba84b8d205faa5c47924c928b0b0074dba/tools/fetch-sponsors.js#L249-L255

I think we can now remove this entry from `donations.json` as it's already in `sponsors.json` and that alone will probably fix the build:

https://github.com/eslint/eslint.org/blob/415e0bba84b8d205faa5c47924c928b0b0074dba/src/_data/donations.json#L254-L262

Reply:
> Can we update the data files in this PR instead?

Absolutely! It's the best workaround actually.